src/DirectWebServices/Shared/DWSConverters/RegionExposure/RegionExposureConverter.ts

```diff
@@ -0,0 +1,228 @@
+/* *
+ *
+ *  (c) 2009-2025 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Jedrzej Ruta
+ *
+ * */
+
+
+'use strict';
+
+
+/* *
+*
+*  Imports
+*
+* */
+
+
+import type { RegionExposureOptions } from './RegionExposureOptions';
+import MorningstarConverter from '../../../../Shared/MorningstarConverter';
+
+/**
+ *
+ * Constants
+ *
+ */
+
+
+const PREFIXES = {
+    Equity: 'equityRegion',
+    FixedIncome: 'fixedIncRegion',
+    FixedIncomeGeographic: 'fixdIncGeographic',
+    RevenueExposure: 'revenueExposureByRegionPerc'
+};
+
+const GEO_SUBPREFIXES = [
+    'DevelopmentStatusBreakdown',
+    'SuperRegionBreakdown',
+    'PrimaryRegionBreakdown',
+    'RegionBreakdown'
+] as const;
+
+const REGIONS: Record<string, string> = {
```

Comment:
Interesting idea!
I have mixed filings though:

1. Are you sure that `AsiaEmrg` and `AsiaEmerging` are exactly the same? And shouldn't use separate cells/fields in DataTable? The same about others (ExEur and ExEuro etc). 
2. Other converters don't include ID -> name translations. I'm not sure whether l18n to english should be in the connector, or in a demo.
      1. Reason: If they introduce new mapping/region, we will end up with mixed regions' names, until we release new version. Or fields will be missing in data table.